sc-deployer/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@massalabs/massa-sc-deployer",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "test": "ts-node tests/index.ts",
+    "build-package": "rm -rf dist/* && tsc -d -p ./tsconfig.json && cp deployer.wasm dist/",
```

Comment:
we should add a command to build the wasm files (deployer, and test) and remove from the repository

Reply:
The command is the build above. Removed from repository.